docs/man/man1/dmd.1

```diff
@@ -1,4 +1,4 @@
-.TH DMD 1 "2011-02-22" "Digital Mars" "Digital Mars D"
+.TH DMD 1 "2011-02-22" "D Language Foundation" "D Language Foundation"
```

Comment:
I think the precise wording should be "the D Language Foundation".  See the bottom of http://dlang.org.

Reply:
Correct, our brand includes the "The".

Reply:
Ok. Should I just replace it for the man page or for all occurrences?

Reply:
For considering, I say all.

Reply:
Yes, all.